src/test/java/com/epam/tc/hw5/ex1/cucumber/hooks/CucumberHook.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.tc.hw5.ex1.cucumber.hooks;
+
+import com.epam.tc.hw5.ex1.cucumber.context.TestContext;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class CucumberHook {
```

Comment:
you don't have to create separated CucumberHook classes for each exercise.
fix it.